calendar_event_report/report/calendar_event_print_template.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="calendar_event_report.event_report_view">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="calendar_event_report.calendar_event_report_view" t-lang="o.partner_id.lang"/>
+
+            </t>
+        </t>
+    </template>
+
+    <template id="calendar_event_report.calendar_event_report_view">
+        <t t-set="o" t-value="o.with_context({'lang':o.partner_id.lang})"/>
+        <t t-call="web.basic_layout">
+            <div class="page">
+                <h2 class="h3-report">
+                    <t t-if="o.allday">
+                        <span t-esc="o.get_formatted_date(o.start_date, 'start_date', 1)"/>
+                    </t>
+                    <t t-else="">
+                        <span t-esc="o.get_formatted_date(o.start_datetime, 'start_datetime', 1)"/>
+                    </t>
+                </h2>
+                <p>
+                    <h2>
+                        <span t-field="o.room_id"/>
```

Comment:
replace with resource_id so we can replace the project_resource_caelndar dependency with calendar_resource.
And at the same time we need to create a new branch in gitlab to install this module and add replace resource_id with room_id in project specific